extensions/CheckUser/CheckUser.hooks.php

```diff
@@ -39,7 +39,7 @@ public static function updateCheckUserData( RecentChange $rc ) {
 			'cuc_title'      => $rc_title,
 			'cuc_minor'      => $rc_minor,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$rc_minor" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)

[project-id]: wikia:app